### PR TITLE
Echo a message on setting PKG_CONFIG_PATH

### PR DESCRIPTION
**Pull request by wantehchang**
Similar to the message added to third-party/svt.cmd in commit 1a1b4d4d.

### CODE REVIEW

**Comment by wantehchang** [comment]
@farindk Dirk: Please review. Would you like me to add the message to other third_party/*.cmd files?

File: third-party/aom.cmd [hunk 1]
```diff
@@ -18,3 +18,8 @@ cmake -S . -B build.libavif -G Ninja -DCMAKE_INSTALL_PREFIX="$(pwd)/dist" -DCMAK
 ninja -C build.libavif
 ninja -C build.libavif install
 cd ..
+
+echo ""
+echo "----- NOTE ----"
+echo "Please add the path to the pkg-config file to your PKG_CONFIG_PATH, like this:"
+echo "export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:`pwd`/aom/dist/lib/pkgconfig"
```
Comment (bradh) [inline third-party/aom.cmd:25]:
```suggestion
echo "export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:$(pwd)/aom/dist/lib/pkgconfig"
```

for consistency with the -CMAKE_INSTALL_PREFIX approach

File: third-party/dav1d.cmd [hunk 2]
```diff
@@ -21,3 +21,8 @@ meson build --default-library=static --buildtype release --prefix "$(pwd)/dist"
 ninja -C build
 ninja -C build install
 cd ..
+
+echo ""
+echo "----- NOTE ----"
+echo "Please add the path to the pkg-config file to your PKG_CONFIG_PATH, like this:"
+echo "export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:`pwd`/dav1d/dist/lib/x86_64-linux-gnu/pkgconfig"
```
Comment (bradh) [inline third-party/dav1d.cmd:28]:
```suggestion
echo "export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:$(pwd)/dav1d/dist/lib/x86_64-linux-gnu/pkgconfig"
```

**Review by bradh — COMMENTED** [review]
The general approach looks fine.

File: third-party/aom.cmd (hunk 1 above)
Comment (wantehchang) [reply third-party/aom.cmd]: Done

File: third-party/dav1d.cmd (hunk 2 above)
Comment (wantehchang) [reply third-party/dav1d.cmd]: Done.

**Review by wantehchang — COMMENTED** [review]
Thanks for the review!

File: third-party/aom.cmd [hunk 3]
```diff
@@ -18,3 +18,8 @@ cmake -S . -B build.libavif -G Ninja -DCMAKE_INSTALL_PREFIX="$(pwd)/dist" -DCMAK
 ninja -C build.libavif
 ninja -C build.libavif install
 cd ..
+
+echo ""
+echo "----- NOTE ----"
+echo "Please add the path to the pkg-config file to your PKG_CONFIG_PATH, like this:"
+echo "export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:$(pwd)/aom/dist/lib/pkgconfig"
```
Comment (wantehchang) [inline third-party/aom.cmd:25]: This path works on all platforms.

File: third-party/dav1d.cmd [hunk 4]
```diff
@@ -21,3 +21,8 @@ meson build --default-library=static --buildtype release --prefix "$(pwd)/dist"
 ninja -C build
 ninja -C build install
 cd ..
+
+echo ""
+echo "----- NOTE ----"
+echo "Please add the path to the pkg-config file to your PKG_CONFIG_PATH, like this:"
+echo "export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:$(pwd)/dav1d/dist/lib/x86_64-linux-gnu/pkgconfig"
```
Comment (wantehchang) [inline third-party/dav1d.cmd:28]: Hi Brad,

I copied this path from the comment at line 4. This path will only work on Linux x86_64. I can clarify that in the message. How about this?

> Please add the path to the pkg-config file to your PKG_CONFIG_PATH. For example, on Linux x86_64, run:
> export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:$(pwd)/dav1d/dist/lib/x86_64-linux-gnu/pkgconfig

Comment (bradh) [reply third-party/dav1d.cmd:28]: That seems better.

Comment (wantehchang) [reply third-party/dav1d.cmd:29]: Done.

**Review by bradh — APPROVED** [review]
LGTM

**Comment by wantehchang** [comment]
@farindk Dirk: Please review. Thanks!

**Comment by farindk** [comment]
Thank you @wantehchang and @bradh

**Pull request merged**